### ISSUE TITLE
Can we use TLS for LPR print ? 

### ISSUE THREAD

**Issue by sim789** [MaxxtonGroup/internet-printer-library#3]
Is that any possible for LPR use TLS ?

**Comment by robsonke** [comment]
It's not part of the library but shouldn't be too hard to add, you'll probably need an SSLSocket instead of a Socket.

**Comment by sim789** [comment]
@robsonke 
I used SSLSocket instead of a Socket and when i use SSLSocket exception throw with printer offline. when i use  Socket  its works.
any Example if you can provide.

**Comment by robsonke** [comment]
No, to be honest it's a while ago we built this library and our communication with those printers is pretty simple. Sounds logical to me that an offline printer gives exceptions when you try to connect to it though.
